Bottom: 3f6b82f5b0a388b14942252848948fda5761418c
Top:    0e03b924581c8db149a52ad5acd30d97b4bb9180
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 11:25:05 +0000

Refresh of ripple-out-step-emission-2

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 89721b5..8f35ca1 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -63,7 +63,6 @@ object3d_test_init (void)
 {
   //object3d_test_object = object3d_create_test_cube ();
   object3d_test_object = object3d_from_board_outline ();
-  object3d_test_board_outline ();
 }
 
 object3d *
@@ -503,16 +502,6 @@ object3d_from_board_outline (void)
   return object;
 }
 
-void
-object3d_test_board_outline (void)
-{
-  object3d *board_outline;
-
-  board_outline = object3d_from_board_outline ();
-  object3d_export_to_step (board_outline, "object3d_test.step");
-  destroy_object3d (board_outline);
-}
-
 object3d *
 object3d_from_tracking (void)
 {
